### PR TITLE
[DNM] Uplink Changes

### PR DESCRIPTION
**Pull request by Mark9013100**
<s>Need to make a changelog entry</s> and get some feedback codewise.

Forum topic: http://ss13polaris.com/forums/viewtopic.php?f=18&t=759

* Traitors, and any antag with an uplink starts with 24 telecrystals.

* Prices changed across the board, weapons are more pricey, utility items cost less.

* Adds stealthy versions of the ablative, riot, and bullet-resistant armors and helmets.

* Adds the EMT, Combat, and tool-belt.

* Adds the combat turtleneck and tactical mask to the combat armor kit.

* Adds the blood-red voidsuit.

* Adds sunglasses, obvious thermals, night-vision goggles, and superior welder goggles.

* Adds the Energy shield, binoculars, standard mining drill, black insulated gloves, and the syndicate dufflebags.

* Removes duplicate chameleon projector.

Plus some minor edits to the ushanka, and warden's jacket.

### CODE REVIEW

**Comment by Yoshax** [comment]
An actual price comparison beyond X costs 4 and now costs 5 would be nice.

**Comment by Yoshax** [comment]
And why 20? It makes it far harder to draw a simple comparison between before and after, doubling the amount if you wanted to allow for more price variance would have made more sense imo, and allow for simple comparisons to be made, as the TC amount be 2x not 1.67x

File: code/datums/uplink/tools.dm [hunk 1]
```diff
@@ -78,8 +109,53 @@
...
+	name = "Mining Drill"
+	item_cost = 4
+	path = /obj/item/weapon/pickaxe
+
+/datum/uplink_item/item/tools/syndieduffle
+	name = "Dufflebag"
+	item_cost = 4
+	path = /obj/item/weapon/storage/backpack/dufflebag/syndie
+
+/datum/uplink_item/item/tools/drill
```
Comment (Raptor1628) [inline code/datums/uplink/tools.dm]: Think you forgot to change this path.

Comment (Mark9013100) [inline code/datums/uplink/tools.dm]: OH, was abit tired during this, will fix.

**Comment by Spookerton** [comment]
It would probably be more useful to just buff TCs to a large number, like 100, and then increase the cost of buyables multiplicatively, rounding down to the closest integer. As is, the changes here are pretty inconsistent.

**Comment by Mark9013100** [comment]
Alright, I'll bump the max TC to 24 as per Yosh's suggestion and take a look at the prices again.

**Comment by Fox-McCloud** [comment]
>Adds the Energy shield, binoculars, standard mining drill, black insulated gloves, and the syndicate dufflebags.

Do energy shields reflect laser fire on Polaris as they do elsewhere?

**Comment by Neerti** [comment]
They don't reflect, only block.

**Comment by Mark9013100** [comment]
Outline here: http://pastebin.com/a8gE8Jqy
Will work on a price comparison later on.

**Comment by Mark9013100** [comment]
Closing because I don't see myself finishing this for awhile, will attempt again in the future.

**Pull request closed without merging**